com.saleae.Logic.json

```diff
@@ -0,0 +1,83 @@
+{
+    "app-id": "com.saleae.Logic",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "18.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "logic-wrapper",
+    "finish-args": [
+        "--socket=x11",
+        "--share=ipc",
+        "--device=all",
+        "--env=LD_LIBRARY_PATH=/app/extra/lib",
+        "--env=QT_ACCESSIBILITY=0",
+        "--env=NO_AT_BRIDGE=1"
+    ],
+    "modules": [
+        {
+            "name": "Logic",
+            "buildsystem": "simple",
+            "sources": [
+                {
+                    "type": "file",
+                    "path": "com.saleae.Logic.desktop"
+                },
+                {
+                    "type": "file",
+                    "path": "com.saleae.Logic.appdata.xml"
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html